### DISCUSSION TITLE
HRR Decreases

### DISCUSSION THREAD

**Discussion by SXFire12** [firemodels/cfast#2132] [Q&A] [answered]
Dear All, I have a question regarding the HRR in CFAST. I expect that the generated HRR would be equal to the input. However, in my case is different. There are four openings to the outside. I also changed the lower oxygen limit but it does not make any difference. What is your opinion on solving this problem?

[Rooms.zip](https://github.com/user-attachments/files/16709638/Rooms.zip)
![HRR CFAST](https://github.com/user-attachments/assets/77645ba6-e141-49d4-89fe-8e309ef76bd1)

**Comment by mcgratta** [comment]
I'm not sure exactly how CFAST works, but in FDS, even if you turn off the empirical extinction model, you might still be in a situation where the oxygen in the room is completely depleted and combustion cannot occur. The lower oxygen limit tells CFAST to stop consuming oxygen when its concentration falls below that value. If the LOL is zero, CFAST will try to consume all available oxygen. If there is no oxygen at all, the combustion should stop.

**Reply by SXFire12** [reply]
In my case now, there is oxygen supplied from the four different openings. My concern is why the generated HRR decreases while there are sufficient supply of oxygen. I ran the same case in FDS, and I got the equal HRR that I wanted. I tried to change the area of fire in CFAST, but it seems that was not the solution.

**Comment by drjfloyd** [comment]
FDS and CFAST are very different codes. You have a small room with an under
ventilated fire connected to a larger room with more ventilation.  You are
probably pushing the bounds of the types of scenarios CFAST is best at.
Also FDS reports the HRR over the whole domain whereas CFAST does
compartment by compartment. Are you comparing equivalent outputs?

**Reply by SXFire12** [reply]
I'm not sure If I compare equivalent outputs. I thought that the HRR from CFAST was for the whole domain as well. My intention now is to compare expected and actual HRR from CFAST. I tried to simulate with the lower HRR (1500 kW) and both the expected and actual are the same. When it comes to higher HRR ( > 2000 kW), the HRR suddenly decreases after reaching the peak.

**Comment by preneke** [comment]
Can you include your actual case? Also can you see if there is a door jet fire or vent fire in the output.

**Comment by preneke** [comment]
Sorry, it is early in the morning for me. I missed the file.

**Comment by preneke** [comment, marked as answer]
All or almost all the fuel is being burned. For clarity, the case I have only has one vent into Comp 6 from Comp 7. The plot shows that the fuel not being burned in Comp 6 is being burned in a vent fire in Comp 27, except for a small amount burning outside from the vents connecting to the outside. 
![image](https://github.com/user-attachments/assets/97872781-380f-44be-868d-9c3a4fe14847)

**Reply by SXFire12** [reply]
Thanks for your explanation, but why is all the fuel not being burned in Comp 6?

**Comment by drjfloyd** [comment]
Do a hand calculation for the maximum sustained ventilation flow through
the door opening you defined. How much methane would that amount of oxygen
support.

**Comment by SXFire12** [comment]
Thanks all. It is becoming clear now. I really appreciate your support
